src/ai/agentzero/AgentZeroClient.ts

```
@@ -0,0 +1,1067 @@
+                
+                try {
+                    // Try to parse error response as JSON
+                    errorObj = JSON.parse(errorText);
+                } catch (e) {
+                    // If not JSON, use the raw text
+                    errorObj = { message: errorText };
+                }
+                
+                // Check for credit limit errors from OpenRouter
+                const isCreditLimitError = 
+                    (errorObj.error?.message && errorObj.error.message.includes('credits are required')) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)